lib/context_adapter_server.js

```diff
@@ -25,6 +25,7 @@
 
 var caLogger = require('logops');
 var caConfig = require('./context_adapter_configuration');
+caLogger.setLevel(caConfig.LOGOPS_LEVEL);
```

Comment:
Same as above. And more interesting: do you realize that the same `setLevel()` function is being executed multiple times?